gluten-ut/spark34/src/test/scala/io/glutenproject/utils/velox/VeloxTestSettings.scala

```diff
@@ -68,44 +68,14 @@ class VeloxTestSettings extends BackendTestSettings {
     .exclude("partitioned join: number of buckets mismatch should trigger shuffle")
     .exclude("partitioned join: only one side reports partitioning")
     .exclude("partitioned join: join with two partition keys and different # of partition keys")
+    // disable as both checks for SMJ node
     .excludeByPrefix("SPARK-41413: partitioned join:")
-    .exclude("SPARK-42038: partially clustered: with different partition keys and both sides partially clustered")
-    .exclude("SPARK-42038: partially clustered: with different partition keys and missing keys on left-hand side")
-    .exclude("SPARK-42038: partially clustered: with different partition keys and missing keys on right-hand side")
-    .exclude("SPARK-42038: partially clustered: left outer join")
-    .exclude("SPARK-42038: partially clustered: right outer join")
-    .exclude("SPARK-42038: partially clustered: full outer join is not applicable")
-    .exclude("SPARK-42038: partially clustered: with dynamic partition filtering")
+    .excludeByPrefix("SPARK-42038: partially clustered:")
   enableSuite[GlutenLocalScanSuite]
   enableSuite[GlutenMetadataColumnSuite]
   enableSuite[GlutenSupportsCatalogOptionsSuite]
   enableSuite[GlutenTableCapabilityCheckSuite]
   enableSuite[GlutenWriteDistributionAndOrderingSuite]
-    .exclude("ordered distribution and sort with same exprs: append")
-    .exclude("ordered distribution and sort with same exprs: overwrite")
-    .exclude("ordered distribution and sort with same exprs: overwriteDynamic")
-    .exclude("clustered distribution and sort with same exprs: append")
-    .exclude("clustered distribution and sort with same exprs: overwrite")
-    .exclude("clustered distribution and sort with same exprs: overwriteDynamic")
```

Comment:
@ulysses-you These (GlutenWriteDistributionAndOrderingSuite) are the one which were passing without any changes, so I just removed them from exclude